test/unit/spec/narrative_core/kbaseNarrativeAppPanel-spec.js

```diff
@@ -33,7 +33,7 @@ define([
                         subtitle: 'Run an app',
                         tooltip: 'Run an app',
                         icon: {
-                            url: 'img?method_id=a_module/an_app&image_name=an_app.png&tag=release',
+                            isApp: true,
```

Comment:
prevent the call out to the KBase servers -- this just generates a generic app icon